databases/psql_replication.rst

```diff
@@ -0,0 +1,120 @@
+Postgres Replication
+====================
+
+What to do if Replication Streaming on Postgres fails
+-----------------------------------------------------
+
+Important! Read this section first before doing anything!
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this section, very powerful commands are described. So before execute anything, be 100% sure that you are on the right server!
+Else you could cause a lot of damage. 
+
+#. To check if you are on the right server use the follwing query.
+#. On master the result is empty. On slave it is not (see example). 
+
+.. code:: 
+
+    SELECT pg_last_xlog_receive_location();
+       
+      pg_last_xlog_receive_location 
+     -------------------------------
+          2493/54AAF118
+
+
+Check if Replication is working
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+First of all you shuld check the delay. If the delay is critical nagios or Backoffice will inform you about it.
+This will look like this:
+
+.. code::
+
+   POSTGRES_HOT_STANDBY_DELAY CRITICAL: DB postgres (host:db01slave) 147866242312 seconds
+
+if the number (delay in seconds) is extremely high then you know something must be wring with the replication.
+
+#. Another way to check the delay directly is:
+
+.. code::
+
```

Comment:
I'd add, "check if your on slave" as one of the steps. You can just link the section further up.

Reply:
done